### PR TITLE
Update core_packages.txt

### PR DESCRIPTION
**Pull request by supechicken**

### CODE REVIEW

**Comment by supechicken** [comment]
> Thank you for the update @supechicken. Can you please sort the packages in alphabetical order? This is so we can avoid duplicates.

Done

File: tools/core_packages.txt [hunk 1]
```diff
@@ -1,74 +1,104 @@
...
+libssh
 libssp
 libtasn1
 libtirpc
+libunbound
 libunistring
 libxml2
+libxslt
 libyaml
+llvm
```
Comment (uberhacker) [inline tools/core_packages.txt:61]: This is in core now?  Are you sure?  If it is, we need to remove it immediately.

File: tools/core_packages.txt [hunk 2]
```diff
@@ -1,74 +1,104 @@
...
 py3_pip
 py3_setuptools
 py3_wheel
 python2
 python3
 readline
+rsync
 rtmpdump
 ruby
-sed
```
Comment (uberhacker) [inline tools/core_packages.txt:67]: This should be core.  The out-of-the-box sed is worthless.

File: tools/core_packages.txt [hunk 3]
```diff
@@ -1,74 +1,104 @@
...
+docbook_xsl_nons
 e2fsprogs
 elfutils
 expat
 filecmd
 flex
-gdbm
+gcc
 gcc11
+gdbm
```
Comment (uberhacker) [inline tools/core_packages.txt:23]: Good call.  gdbm was out of sequence.

File: tools/core_packages.txt (hunk 1 above)
Comment (supechicken) [reply tools/core_packages.txt:61]: The list is generated by `crew deps core`, so it might be correct?

File: tools/core_packages.txt (hunk 2 above)
Comment (supechicken) [reply tools/core_packages.txt:67]: AFAIK `sed` was moved away to `buildessential` times ago (only `patch` and `build` section needs it)

File: tools/core_packages.txt [hunk 4]
```diff
@@ -1,74 +1,104 @@
-acl
-attr
-binutils
```
[the remark this answers is not in the document]
Comment (supechicken) [reply tools/core_packages.txt:3]: See #6642

File: tools/core_packages.txt [hunk 5]
```diff
@@ -1,74 +1,104 @@
...
 brotli
 bz2
-c_ares
 ca_certificates
+c_ares
```
[the remark this answers is not in the document]
Comment (supechicken) [reply tools/core_packages.txt:4]: Seems the `sort` command doesn't think so 😅

[the remark this answers is not in the document]
Comment (supechicken) [reply tools/core_packages.txt:3]: Fixed

File: tools/core_packages.txt (hunk 2 above)
Comment (uberhacker) [reply tools/core_packages.txt:67]: And how many packages use it?  About half?  I think we should add it back.

File: tools/core_packages.txt (hunk 1 above)
Comment (uberhacker) [reply tools/core_packages.txt:61]: We need to remove it then.  There should be no reason to include this in core.  Not only is it massive but it's only needed for builds so it really should be in buildessential.

File: tools/core_packages.txt (hunk 2 above)
Comment (uberhacker) [reply tools/core_packages.txt:67]: After further thought, it's fine to leave this in buildessential.  I think we should move llvm to buildessential, if possible.  That should NOT be in core.

Comment (satmandu) [reply tools/core_packages.txt:67]: > After further thought, it's fine to leave this in buildessential. I think we should move llvm to buildessential, if possible. That should NOT be in core.

As soon as we can break out the llvm libraries into another package!

Comment (uberhacker) [reply tools/core_packages.txt:67]: Requiring llvm in core is cringeworthy.

**Comment by uberhacker** [comment]
I did a fresh install and added missing packages from this PR.  Packages I would like to make an effort to remove from core and ideally into buildessential instead in the future include:
```
autoconf
automake
cmake
gcc11
jdk8
llvm
make
patch
patchelf
pkgconfig
valgrind
```

**Pull request merged**